index.html

```diff
@@ -15,12 +15,43 @@
 <body>
 
 <header>
+    <nav>
+
+    </nav>
 </header>
 
 <main>
 
     <h1> Weather  </h1>
 
+<!-- City Selector -->
```

Comment:
I was a little confused with your radio buttons..  You have to select the one already selected to make it switch?

Reply:
Hi Suzy, I am using chrome and not experiencing this issue. Would you be able to tell me what I could do to reproduce the issue?  Thanks!

Reply:
It is now working, not sure why I had the issue before.  My bad!    All looks good!